### PR TITLE
Change to multiple MAC in one BTDEVADDR

### PR DESCRIPTION
**Pull request by hanhsuan**
Since CQT-2031 has merged into checkbox and released  in 2.6, we could change the default palinbox.conf of oem-qa-tools to multiple MAC in one BTDEVADDR style.

### CODE REVIEW

**Review by clairlin53 — APPROVED** [review]
LGTM

File: Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf [hunk 1]
```diff
@@ -15,9 +15,7 @@ OPEN_N_SSID = cert-n-open-tel-l3-01
 OPEN_AC_SSID = cert-ac-open-tel-l3-01
 OPEN_AX_SSID = cert-ax-open-tel-l3-01
 # === Select one BTDEVADDR below ===
```
Comment (LiaoU3) [inline Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf:17]: Maybe we don't need this line after we can automatically select bluetooth address?

File: Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf [hunk 2]
```diff
@@ -15,9 +15,7 @@ OPEN_N_SSID = cert-n-open-tel-l3-01
...
-BTDEVADDR = 7C:B2:7D:4B:14:95
-# BTDEVADDR = 28:3A:4D:46:79:C0
-# BTDEVADDR = 34:6F:24:A8:93:EE
+BTDEVADDR = 7C:B2:7D:4B:14:95,28:3A:4D:46:79:C0,34:6F:24:A8:93:EE,80:32:53:D8:0D:1E
 # ==================================
```
Comment (LiaoU3) [inline Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf:19]: And also this line.

**Review by LiaoU3 — CHANGES_REQUESTED** [review]

File: Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf [hunk 3]
```diff
@@ -14,10 +14,6 @@ OPEN_BG_SSID = cert-bg-open-tel-l3-01
 OPEN_N_SSID = cert-n-open-tel-l3-01
 OPEN_AC_SSID = cert-ac-open-tel-l3-01
 OPEN_AX_SSID = cert-ax-open-tel-l3-01
-# === Select one BTDEVADDR below ===
-BTDEVADDR = 7C:B2:7D:4B:14:95
-# BTDEVADDR = 28:3A:4D:46:79:C0
-# BTDEVADDR = 34:6F:24:A8:93:EE
-# ==================================
+BTDEVADDR = 7C:B2:7D:4B:14:95,28:3A:4D:46:79:C0,34:6F:24:A8:93:EE,80:32:53:D8:0D:1E
```
Comment (LiaoU3) [inline Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf:17]: @hanhsuan Quick question, is this address `80:32:53:D8:0D:1E` the new one?

Comment (hanhsuan) [reply Tools/PC/oem-qa-checkbox-installer/conf/plainbox.conf:17]: That is the Lab-office one

**Review by LiaoU3 — APPROVED** [review]
LGTM <3

**Pull request merged**